### PR TITLE
remove dependency on deprecated makara-writer-amd and add unit test

### PR DESCRIPTION
**Pull request by grawk**
- adds an actual unit test
- removes dependency on `makara-writer-amd` as that module is being deprecated with other refactoring of the `makara-*` modules

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,7 +30,9 @@
   },
   "homepage": "https://github.com/krakenjs/construx-makara-amdify#readme",
   "dependencies": {
-    "makara-writer-amd": "git://github.com/grawk/makara-writer-amd#master"
+    "iferr": "^0.1.5",
+    "makara-writer-amd": "git://github.com/grawk/makara-writer-amd#master",
```
Comment (subeeshcbabu-zz) [inline package.json]: `makara-writer-amd` not published yet?

Comment (grawk) [inline package.json]: oops! forgot to remove the dependency from package.json. fixed.

**Comment by subeeshcbabu-zz** [comment]
👍

File: index.js [hunk 2]
```diff
@@ -17,8 +17,14 @@
  \*───────────────────────────────────────────────────────────────────────────*/
 'use strict';
 
-var amdBuilder = require('makara-writer-amd').amdBuilder;
-
+var path = require('path');
+var iferr = require('iferr');
+var spundle = require('spundle');
+var moduleBuilder = function (appRoot, m, cb) {
+    spundle(path.resolve(appRoot, 'locales'), m[2], m[1], iferr(cb, function (out) {
```
Comment (skoranga) [inline index.js:24]: some sanity check on `m`?

Comment (grawk) [inline index.js:24]: Thanks. I added a sanity check in the main returned function below for the locale parts

**Comment by shaunwarman** [comment]
👍

**Comment by skoranga** [comment]
👍

**Pull request merged**